### PR TITLE
Upgrading tika to 2.6.0

### PR DESCRIPTION
**Pull request by masseyke**
Upgrading tika and its dependencies.

### CODE REVIEW

**Comment by masseyke** [comment]
@elasticmachine run elasticsearch-ci/part-2

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -2831,56 +2841,111 @@
...
             <sha256 value="361c615f0c5322ba786d2b06274f9946f5ec8de7784edf4fa26375f755299a45" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.tika" name="tika-parser-xmp-commons" version="2.6.0">
+         <artifact name="tika-parser-xmp-commons-2.6.0.jar">
+            <sha256 value="b96bb7d066ced0a88cf3223363642dcba3d581592d5b0770ed197e7520f5bbf0" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.apache.tika" name="tika-parser-zip-commons" version="2.4.0">
          <artifact name="tika-parser-zip-commons-2.4.0.jar">
```
Comment (jakelandis) [inline gradle/verification-metadata.xml:2940]: Can you remove the elder versions from this file (assuming they are no longer needed)

**Review by jakelandis — APPROVED** [review]
LGTM

**Comment by masseyke** [comment]
A note on this one (because it confused me): `org.apache.pdfbox:jempbox` is versioned separately from the rest of `org.apache.pdfbox`. That is because jempbox was removed from pdfbox as of 2.0 (https://pdfbox.apache.org/2.0/migration.html), but it is still pulled in directly by tika (https://github.com/apache/tika/blob/2.6.0/tika-parent/pom.xml#L344). So I've changed the version to match the one pulled in by tika 2.6.0.

**Review by jbaiera — APPROVED** [review]
:shipit:

**Comment by masseyke** [comment]
@elasticmachine update elasticsearch-ci/part-1

**Comment by masseyke** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Comment by masseyke** [comment]
@elasticmachine run elasticsearch-ci/part-1

**Pull request merged**